src/common/interProcessMessageSender.ts

```diff
@@ -41,12 +41,20 @@ export class InterProcessMessageSender implements InterProcessMessageSender {
             body += data;
         });
 
-        const failPromise = () =>
-            deferred.reject(ErrorHelper.getInternalError(InternalErrorCode.ErrorWhileProcessingMessageInIPMSServer,
-                ExtensionMessage[message]));
+        const failPromise = (reason?: Error & {code?: string}) => {
```

Comment:
The default `Error` type does not have a `code` property, even though a lot of errors actually do, including the `"ENOENT"` error that I'm checking for here.